TODO

```diff
@@ -0,0 +1,10 @@
+[x] Make department filter not required
```

Comment:
If you're in the habit of using a personal TODO file, that's fine - please put it in your personal .gitignore, particular for OS repos.